build.gradle

```diff
@@ -31,10 +31,11 @@ dependencies {
     implementation 'net.sf.squirrel-sql.thirdparty-non-maven:java-cup:0.11a'
     implementation 'org.xerial:sqlite-jdbc:3.43.2.1'
     implementation 'org.apache.logging.log4j:log4j-api:2.21.1'
+    implementation 'commons-cli:commons-cli:1.6.0'
+    implementation 'org.apache.logging.log4j:log4j-core:2.21.1'
```

Comment:
I am not against this change, but I am a bit surprised it was needed.  The library uses the logging interface and rely on the caller to provide a logging implementation.

I assume you added this for your new CLI class, which is fine. But that class doesn't directly log anything.  

What required the addition of this library?

Reply:
I think this is a remnant of when I was trying to resolve a minor slf4j warning. I will remove the dependency.

Reply:
I added the log4j-core dependency back after implementing log4j2 output for the CLI.